vividus-plugin-web-app/src/main/java/org/vividus/steps/ui/web/AlertSteps.java

```diff
@@ -32,19 +34,28 @@
 @TakeScreenshotOnFailure
 public class AlertSteps
 {
-    @Inject private IAlertActions alertActions;
+    private static final String ALERT_PRESENT = "An alert is present";
+    private final IAlertActions alertActions;
 
-    @Inject private IBaseValidations baseValidations;
+    private final IBaseValidations baseValidations;
+
+    private final ISoftAssert softAssert;
```

Comment:
can you please fix formatting a bit in the following way:

static var

object final var
object final var
object final var

so to keep them separate

Reply:
fixed